packages/components/src/components/brandshape/brandshape.stories.ts

```diff
@@ -125,4 +125,23 @@ export const Shapes = {
   `
 };
 
+/**
+ * Use the `object-position` CSS property to adjust the focal point of the image.
+ */
+
+export const ImageTranslateY = {
+  name: 'Image Translation',
+  render: () => html`
+    <sd-brandshape variant="image">
+      <div class="slot slot--border slot--text h-8 w-full">Default slot</div>
+      <img
+        slot="image"
+        src="./placeholders/images/architecture.jpg"
+        style="object-position: top right"
```

Comment:
NIT: instead of setting the style inline, I think you could use a tailwind class `object-right-top`

Reply:
I think the idea in this story specifically was to highlight the usage of the object-position style.